__tests__/address-info.test.js

```diff
@@ -1,11 +1,21 @@
 import TestUtils from './test-utils';
 
+const walletId = 'stub_address_info';
+
 describe('address-info api', () => {
+  beforeAll(async () => {
+    await TestUtils.startWallet({ walletId });
+  });
+
+  afterAll(async () => {
+    await TestUtils.stopWallet({ walletId });
+  });
```

Comment:
Any test that uses a wallet should start and cleanup it's own wallet. The words used for all wallets will be the same but the `walletId` should be unique to each test.